lib/rmt/cli/base.rb

```diff
@@ -67,6 +67,11 @@ def handle_exceptions
         "The SCC credentials are not configured correctly in '/etc/rmt.conf'. You can obtain them from https://scc.suse.com/organization",
         RMT::CLI::Error::ERROR_SCC
       )
+    rescue RMT::Lockfile::ExecutionLockedError
+      raise RMT::CLI::Error.new(
+        "Process is locked. Please check lockfile at #{RMT::Lockfile::LOCKFILE_LOCATION}\n",
+        RMT::CLI::Error::ERROR_OTHER
```

Comment:
This is not a full review, just a question about this message.
It says "check lockfile at (...)". What is a user expected to do with the lock file? How is it helpful to check that file?

Reply:
If that lock file exists and there's no RMT command running, we have big trouble...
When zypper is locked, it looks like this, which I think is more useful:

```
> sudo zypper in xfce
System management is locked by the application with pid 6096 (/usr/bin/ruby).
Close this application before trying again.
```

We could say "Close this application or wait for it to finish before trying again" instead?

Reply:
:+1: updated. new messages looks like:
'Process is locked by the application with pid 19032. Close this application or wait for it to finish before trying again'